LayoutTests/fast/scrolling/mac/scrollbars/overflow-overlay-scrollbar-reveal.html

```diff
@@ -39,6 +39,12 @@
             const x = scrollerBounds.left + 10;
             const y = scrollerBounds.top + 10;
 
+            // Wait for the backing store to be created
+            await UIHelper.waitForConditionAsync(() => {
```

Comment:
if we just call doTest on a zero delay timer (like most other scrolling tests) can we get rid of UIHelper.waitForConditionAsync ?

Reply:
Updated this to use `await UIHelper.renderingUpdate()`